zgrab2/multiple.go

```diff
@@ -0,0 +1,31 @@
+package zgrab2
+
+import (
+	"errors"
+	"os"
+
+	"github.com/ajholland/zflags"
+)
+
+type MultipleConfig struct {
```

Comment:
I thought we predetermined that a config file **does** necessarily define multiple scans?

And yes, config file override's the global config things. The use cases should be single scan: all options entered through cli. Multiple scan: all options entered through config file, but also supports global config things being entered from cli, however if you specify both (I don't see why) config file takes precedence.